### PR TITLE
chore: updates near-* dependencies to 0.27 release

### PR DESCRIPTION
**Pull request by akorchyn**
@race-of-sloths

### CODE REVIEW

**Comment by race-of-sloths** [comment]
@akorchyn Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/akorchyn?type=bot&pr=near/near-sdk-rs/1254&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/akorchyn?type=bot&pr=near/near-sdk-rs/1254&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/akorchyn)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/akorchyn)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by dj8yfo** [comment]
this looks like a spot where a self-referential struct is moved https://github.com/near/near-sdk-rs/blob/master/near-sdk/src/environment/env.rs?plain=1#L102

**Comment by dj8yfo** [comment]
checked that this doesn't block #1253 in any way with #1255

File: .github/workflows/test.yml
Comment (dj8yfo) [inline .github/workflows/test.yml:17]: would you care to also change these 2 beautiful lines, badge https://github.com/near/near-sdk-rs/blob/eda17d5e2705ec71a1c704b483bae86dd79e53cd/README.md?plain=1#L15 and 
MSRV mention https://github.com/near/near-sdk-rs/blob/eda17d5e2705ec71a1c704b483bae86dd79e53cd/README.md?plain=1#L324

**Comment by dj8yfo** [comment]
@race-of-sloths score 5

**Review by dj8yfo — APPROVED** [review]

File: .github/workflows/test.yml
Comment (akorchyn) [reply .github/workflows/test.yml:17]: Done

**Comment by akorchyn** [comment]
> This looks like a spot where a self-referential struct is moved https://github.com/near/near-sdk-rs/blob/master/near-sdk/src/environment/env.rs?plain=1#L102

I am not sure that I understand what you imply by that :(

**Review by dj8yfo — APPROVED** [review]

**Comment by dj8yfo** [comment]
@akorchyn looks like some glitch, as first commit probably didn't have workflows update. I'll try rerunning Test core job, but you must have perm to rerun too.

**Comment by akorchyn** [comment]
It's not a glitch actually. You specify required job names in repo settings. But as rust toolchain is part of the name, that's why it doesn't match. I don't have enough rights

@frol can you update job config

**Comment by dj8yfo** [comment]
@akorchyn cancel the running jobs, then try closing/reopening the pr.
In #1253 the first pushed commit changed the workflows and they were picked up correctly, not requiring stuff from pr's base branch

**Comment by dj8yfo** [comment]
Or maybe even closing, adding some whitespace change (so that's its a new commit, unknown to gh), and opening a new pr from another branch. I don't know, why gh wants to make everything so painful, but this pr https://github.com/near/near-sdk-rs/pull/1231/files seemingly had a similar change and i don't remember anyone having to edit repo config to merge it.

**Comment by akorchyn** [comment]
I tried squash and open new PR here: https://github.com/near/near-sdk-rs/pull/1258

Seems to be like it doesn't work that way :(

**Pull request merged**
